### ISSUE TITLE
Crash when using Hyperdimensional Relocator

### ISSUE THREAD

**Issue by asakurato** [unanimated/luaegisub#10]
Normally, if you use non-gui macro "Convert clip to fax" or "convert clip to frz" without the clip, the script will give you a message that there is a missing clip on the line. However, if you hotkey said macros and press the hotkey fast enough, the message doesn't appear and it results in a crash. Happens 100% of the time. I'm on build r8903.

**Comment by unanimated** [comment]
What I'm getting is that I can run it twice without the clip, and the third time it hangs.
Through some experiments I've determined that what happens is that the error message doesn't load (and thus the script can't finish). When I place a log event right before it, the log message shows. This means that aegisub.dialog.display isn't loading the GUI for the error message, for some reason. Which means this seems to be an internal Aegisub issue rather than an issue with a script. Which also means I have no clue why it happens. So basically be careful not to run it on lines without a clip (maybe rescan automation after two failed attempts), because I highly doubt anyone will ever fix this or even figure out why it happens, given that pretty much nobody works on Aegisub anymore. (Though you may try your luck with CoffeeFlux. I think he's the only one still occasionally working on it quietly somewhere in cyberspace.)

**Comment by asakurato** [comment]
Yeah, I guessed it was more Aegisub issue rather than HR. I will open an issue there as well.
